autogpt_platform/infra/helm/autogpt-server/values.prod.yaml

```diff
@@ -98,8 +98,8 @@ env:
   ANTHROPIC_API_KEY: ""
   OPENAI_API_KEY: ""
   REDIS_PASSWORD: ""
-  NUM_GRAPH_WORKERS: 10
-  NUM_NODE_WORKERS: 5
+  NUM_GRAPH_WORKERS: 4
+  NUM_NODE_WORKERS: 2
```

Comment:
This seems low; 2*4 = max 8 connections out of the 60 we are allowed (leaving 52 for the rest of the system). Number of node workers doesn't have to be very high but if we can I think we should increase the max number of graph workers.

Reply:
This is from @majdyz , context: https://discord.com/channels/1126875755960336515/1285234928367308821/1294302355885391916